src/server.js

```diff
@@ -1,6 +1,6 @@
 const express = require('express');
 const app = express();
-
+require('express-async-errors');
```

Comment:
You might need to move `require('express-async-errors');` to line 2, above the `const app = express();` line for it to work correctly